documentation.md

```diff
@@ -30,10 +30,10 @@ Goss
 
 Chase 
 
-  * AR Method: 
-  * Description:
-  * Example:
-  * Anything else?:
+  * AR Method: Group
+  * Description: Returns an array of unique objects based on whatever attribute you have passed it.  
+  * Example: Users.group(:name) will return [#<User id: 1, name: bob, state: Texas> #<User id:2, name:Chase, state: Texas>] 
+  * Anything else?: Group is almost always follwed by a .count.    

```

Comment:
I THINK it's typically followed by count, because grouping them all together isn't very useful unless we are trying to find the number of each identical attribute that exists.  It was a tricky one, because every time I looked up "group", they almost, 100% of time used it either AFTER a join, or BEFORE a count.  I asked other people what they thought about it, and no one could give me an answer of why Group would want to be used in isolation, either.